th2_data_services/provider/v5/provider_api/http.py

```diff
@@ -208,13 +224,23 @@ def __create_stream_connection(self, url: str):
 
         response.release_conn()
 
-    # TODO - add DocString (Grigory)
-    def execute_sse_request(self, url: str):
+    def execute_sse_request(self, url: str) -> Generator[dict, None, None]:
+        """Creates SSE connection to server.
+        Args:
```

Comment:
By style guide https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html we should add `\n` after the first string.